woocommerce-services.php

```diff
@@ -1201,14 +1201,62 @@ public function shipping_zone_method_status_toggled( $instance_id, $service_id,
 			}
 		}
 
-		public function add_meta_boxes() {
+		public function should_show_shipping_debug_meta_box( $post ) {
+			$order            = wc_get_order( $post );
+			$shipping_methods = $order->get_shipping_methods();
```

Comment:
The function `should_show_shipping_debug_meta_box()` gets run for all post types, so `$order` can end up being false here ([doc](https://docs.woocommerce.com/wc-apidocs/function-wc_get_order.html)).

Let's either return `false` if `$order` is false, or use `should_show_shipping_debug_meta_box` only if `$post_type` is `shop_order` in `add_meta_boxes()`.